server.js

```diff
@@ -200,14 +200,15 @@ if (process.env.DEBUG_DUMMY_SMTP) {
     },
   };
 } else {
-  console.log("Attempting to get SMTP credentials from environment...");
+  console.log("Attempting to get SMTP config from environment...");
   kSMTPUsername = process.env.SMTP_USERNAME;
   let password = process.env.SMTP_PASSWORD;
+  let SMTP_host = process.env.SMTP_HOST;
+  let SMTP_port = process.env.SMTP_PORT;
```

Comment:
Possibly unrelated, but these are starting to feel confusingly inconsistent. Variables are either `password`, `SMTP_port`, or `kSMTPUsername`. Not sure if they should be `smtpUsername`, `smtpPassword`, `smtpHost`, and `smtpPort` for consistency (and changed globally, in the case of `kSMTPUsername`).

Reply:
Additionally, we need to null check these in the next line.

Reply:
Let's rename these variables to just host and port. They're single-use, we can keep it simple. Also don't forget to null check them in the line below.